### ISSUE TITLE
convert_fendl script broken

### ISSUE THREAD

**Issue by makeclean** [openmc-dev/data#6]
As in #5 CI mayve stopped this, but it seems that convert_fendl is broken - I get this error

```
Traceback (most recent call last):
  File "convert_fendl.py", line 87, in <module>
    downloaded_file = download(url, context=ssl._create_unverified_context())
  File "/home/adavis/opt/openmc/openmc/_utils.py", line 27, in download
    req = urlopen(url, **kwargs)
  File "/usr/lib64/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib64/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib64/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib64/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib64/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib64/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 403: Forbidden

```

When I run. Also could be user error, if it is let me know what Im doing wrong

**Comment by makeclean** [comment]
@Shimwell any ideas?

**Comment by paulromano** [comment]
Looks like IAEA changed something in their configuration such that the download needs to appear as though it's [coming from a browser](https://stackoverflow.com/a/49085731/1572453). Unfortunately, that's not something that normal CI would have caught (unless we were running nightly tests or something). Anyway, I'll put in a fix (this requires something extra in OpenMC's `download` function).

**Comment by makeclean** [comment]
Thats my point, maybe we should be running nightly tests that tries to build and get all the data, each one as a different element in the matrix?

**Comment by paulromano** [comment]
Interesting, I didn't think this was possible with Travis but evidently [it is](https://blog.travis-ci.com/2016-12-06-the-crons-are-here). The problem we'll run into is that the "generate" scripts actually run NJOY for all evaluations which takes a long time (longer than the Travis job time limits).

**Comment by paulromano** [comment]
This was fixed by #7.
